src/gui/transferlistwidget.cpp

```diff
@@ -1142,11 +1155,13 @@ void TransferListWidget::displayListMenu()
     if (addedPreviewAction)
         listMenu->addSeparator();
     if (oneHasMetadata)
-    {
         listMenu->addAction(actionForceRecheck);
-        listMenu->addAction(actionForceReannounce);
-        listMenu->addSeparator();
-    }
+    // We can not force reannounce torrents that are paused/errored/checking/missing files/queued.
+    // We may already have the tracker list from magnet url without metadata.
+    // So we can force reannounce anyway.
+    listMenu->addAction(actionForceReannounce);
+    actionForceReannounce->setEnabled(oneCanForceReannounce);
```

Comment:
You are encouraged to add tooltip to help users understand why it is disabled, for example: https://github.com/qbittorrent/qBittorrent/blob/0269676e204a3c7472bc0cb308655c25f7a737a2/src/gui/transferlistwidget.cpp#L679

So here it will become:
```c++
actionForceReannounce->setEnabled(oneCanForceReannounce);
if (!oneCanForceReannounce)
    actionForceReannounce->setToolTip(tr("..."));
listMenu->addSeparator();
```

Reply:
And you might need to enable tooltip before usage, for example: https://github.com/qbittorrent/qBittorrent/blob/0269676e204a3c7472bc0cb308655c25f7a737a2/src/gui/transferlistwidget.cpp#L645

Reply:
>And you might need to enable tooltip before usage, 

It's already enabled for this menu.